src/T2DMap.cpp

```diff
@@ -2192,8 +2191,18 @@ void T2DMap::paintAreaExits(QPainter& painter, QPen& pen, QList<int>& exitList,
 }
 
 // Work out text for information box, need to offset if room selection widget is present
-void T2DMap::paintMapInfo(const QElapsedTimer& renderTimer, QPainter& painter, const bool showingCurrentArea, QColor& infoColor)
+void T2DMap::paintMapInfo(const QElapsedTimer& renderTimer, QPainter& painter, const int displayAreaId, QColor& infoColor)
 {
+    #if (QT_VERSION >= QT_VERSION_CHECK(5, 14, 0))
```

Comment:
We have the ifdef macros start at the beginning, they don't respect the indending:

```suggestion
#if (QT_VERSION >= QT_VERSION_CHECK(5, 14, 0))
```